src/ChainSafe.Gaming/RPC/Contracts/Contract.cs

```diff
@@ -85,7 +85,7 @@ public async Task<object[]> Call(string method, object[] parameters = null, Tran
 
             parameters ??= Array.Empty<object>();
 
-            var txReq = await PrepareTransactionRequest(method, parameters, overwrite);
+            var txReq = await PrepareTransactionRequestNoGasInfo(method, parameters, overwrite);
```

Comment:
But now you would eliminate all the POST London forks. What I would suggest is to actually do something like this: if providers GetFeeData returns a null, or a specific object inside of it is null, you just return the PrepareTransactionRequestNoGasInfo. That way we're utilizing both functions.